test_suite/cloud/test_aws.py

```diff
@@ -558,8 +558,24 @@ def test_yum_group_install(self, host):
             pytest.skip('Not applicable to Atomic host AMIs')
 
         with host.sudo():
-            assert host.run_test('yum -y groupinstall "Development tools"'), \
-                'Error while installing Development tools group'
+            dev_tools_install_command = 'yum -y groupinstall "Development tools"'
+            result = host.run(dev_tools_install_command)
+
+            if result.failed:
```

Comment:
IIUC it should fail just for BYOS due to missing subscription.
But it should work with PAYG due to rhui respositories.

@narmaku Shall we immediately fail for PAYG?

Or do I miss some context :-)